core/libraries/payment_methods/EE_Gateway.lib.php

```diff
@@ -481,11 +481,11 @@ protected function _format_partial_payment_line_item_desc(EEI_Payment $payment)
      * Gets the name to use for a line item when sending line items to the gateway
      *
      * @deprecated since 4.9.31 instead use $this->_get_gateway_formatter()->formatLineItemName($line_item,$payment)
-     * @param EEI_Line_Item $line_item
+     * @param EE_Line_Item $line_item
      * @param EEI_Payment   $payment
      * @return string
      */
-    protected function _format_line_item_name(EEI_Line_Item $line_item, EEI_Payment $payment)
+    protected function _format_line_item_name(EE_Line_Item $line_item, EEI_Payment $payment)
```

Comment:
```suggestion
    protected function _format_line_item_name(EE_Line_Item $line_item, EEI_Payment $payment): string
```

Reply:
not adding typing tomain EE core parent classes like this just yet because it will facilitate similar changes to all of the child classes else errors will be raised. When we get everything moved over to the Cafe PHP monorepo, then that can be done easily.